stingray/deadtime/fad.py

```diff
@@ -20,32 +20,44 @@ def _get_fourier_intv(lc, start_ind, end_ind):
     ----------
     lc : a :class:`Lightcurve` object
         Input light curve
+
     start_ind : int
         Start index of the light curve chunk
+
     end_ind : int
         End index of the light curve chunk
 
     Returns
     -------
     freq : array of floats
         Frequencies of the Fourier transform
+
     fft : array of complex numbers
         The Fourier transform
+   
     nph : int
         Number of photons in the interval of the light curve
+
+    nbins : int
+        Number of bins in the light curve segment
+
+    meancounts : float
+        The mean counts/bin in the light curve    
     """
     time = lc.time[start_ind:end_ind]
     counts = lc.counts[start_ind:end_ind]
+    meancounts = np.mean(counts)
 
     fourier = scipy.fftpack.fft(counts)
 
     freq = scipy.fftpack.fftfreq(len(time), lc.dt)
     good = freq > 0
 
-    return freq[good], fourier[good], np.sum(counts)
+    nbins = len(time)
+    return freq[good], fourier[good], np.sum(counts), nbins,  meancounts
```

Comment:
Note that meancounts can be calculated from the number of counts and nbins, no need for the explicit mean calculation at line 49